roles/cobbler/templates/snippets/cephlab_packages_rhel

```diff
@@ -1,3 +1,8 @@
 ## {{ ansible_managed }}
+#set os_version = $getVar('os_version','')
```

Comment:
Do you think docs here are a good idea or do we just assume the reader knows enough about cobbler?  It was unclear to me on first read where this came from or what to expect from it.